test/rulesets/packages/SpecialFunctions.jl

```diff
@@ -50,7 +50,7 @@ end
             isreal(x) || continue
 
             Δx, x̄ = randn(2)
-            Δz = (randn(), randn())
+            Δz = Composite{Tuple{Float64, DoesNotExist}}(randn(), Zero())
```

Comment:
Why this change? Is it related to the BLAS fixes?

Reply:
Not related to the problem with BLAS, but it does cause a problem for testing accumulation.
Since you can add to a tuple